server/app/helpers/auth.js

```diff
@@ -0,0 +1,21 @@
+import jwt from 'jsonwebtoken'
+import { config } from 'dotenv'
+
+config();
+
+const secretKey = process.env.SECRET_KEY;
+
+class Auth {
+	static generateMailToken(payload) {
+		return jwt.sign(payload, secretKey, {expiresIn: '365days'})
```

Comment:
Your mail says the link will last 1hr but your token is for 365 days?

Reply:
Yeah, was thinking about that, since the user would already have an account and will verify email later, how long do u think before this link expires?

Reply:
I think 30 mins is ok sef.

Reply:
Ok, let me just leave it at 1hour

Reply:
I have seen websites do 15days

Reply:
Wow. Really?
All right. 1hr is cool